### ISSUE TITLE
New Tab : No Dark Mode

### ISSUE THREAD

**Issue by Moonay-TeamCoffee** [RedHotRojo/Dark-Mode-Screen#15]
Found a new bug in the newest update (1.10.2)

- Any new tab (_like by pressing on a link with middle click, or opening a new tab and then going into a website_) the new tab wont be affected by the dark mode. Dark mode must be turned OFF / ON to be applied to the new tab (_only changing the settings does not bring the dark mode in function in that tab_), but when I do, the setting are set to a gray overlay, not the actual chosen setting, so the settings must be changed slightly to update the actual overlay's color.
- That last thing happens anytime the dark mode is turned OFF / ON, the chosen values are still displayed as I want them, but the actual overlay is grey, and the sliders must be moved slightly to update the overlay.

This was not a problem before the recent update.

System : Debian 12 "_Bookworm_" (_64 bit_)
Browser : Firefox 102.14.0esr (_64 bit_)

**Comment by RedHotRojo** [comment]
Just uploaded a new version. I'll let you close it after testing.

**Comment by Moonay-TeamCoffee** [comment]
Updated to 1.10.3

TLDR : One bug solved, the other still in effect
- Solved the issue when a new tab is opened the overlay was off
- Still present is the issue where a new tab doesnt have the set setting in Dark mode,  I still need to open the menu and adjust a slider a little bit for the overlay to update.

**Comment by Moonay-TeamCoffee** [comment]
Actually this problem became bigger, just realized that it also reset to some grey overlay anytime I reload a tab

**Comment by RedHotRojo** [comment]
Just released a new version, everything should be fixed. Thanks for the reports.

**Comment by Moonay-TeamCoffee** [comment]
Just tested V1.11

- Problem with the new tabs no having Dark mdoe overlay has been fixed
- Problem with the values needing to then be adjusted is also solved
- Problem with reloading tab "reset" the values is also solved

Thank you very much.

**Comment by Moonay-TeamCoffee** [comment]
Just tested V1.11

- Problem with the new tabs no having Dark mdoe overlay has been fixed
- Problem with the values needing to then be adjusted is also solved
- Problem with reloading tab "reset" the values is also solved

Thank you very much.

**Comment by Moonay-TeamCoffee** [comment]
Umm, Well, I did not expect to come back to this, but I found another "_bug_" ...

Sorta bug because it's really minor, but it is a discrepancy compared to how the plugin worked prior to the 1.10.2 update :
- When The plugin is ON, if I open a file from my PC onto the browser by dragging it in the tab space, that file will initially not have the overlay.

**Comment by RedHotRojo** [comment]
Firefox extensions usually can't affect files opened in the browser. This behavior is described (somewhat poorly) in [this Firefox bug report](https://bugzilla.mozilla.org/show_bug.cgi?id=1454760). What kinds of files are you opening, and are you saying that the extensions was affecting files like PDFs? This extensions has never worked on PDF files for me.

**Comment by Moonay-TeamCoffee** [comment]
I'm not sure .pdf files, but it sure worked on images (_no matter their extentions, .png .jpg ..._) and it still works, but with the behavior I described above
This makes me wonder why It doesnt work with.pdf, since it works with images, there might be  possibility with other kinds of files that Firefox is able to display. Or Maybe not as the bugzilla report you've sent might be affecting .pdf files and others.

**Comment by RedHotRojo** [comment]
Interesting. It should be fixed with the new update.

**Comment by Moonay-TeamCoffee** [comment]
Confirmed, the images are being displayed with the overlay ON, all other behavior seems OK
Doesnt Work with .pdf files, but I didn't expect that to function.
